examples/notificationdemo/source/src/org/kaaproject/kaa/demo/kaanotification/KaaNotificationApp.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2014 CyberVision, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kaaproject.kaa.demo.kaanotification;
+
+import android.app.Activity;
+import android.app.Application;
+import android.content.Context;
+import android.util.Log;
+import android.view.Gravity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.PopupWindow;
+import android.widget.TextView;
+
+import org.kaaproject.kaa.client.AndroidKaaPlatformContext;
+import org.kaaproject.kaa.client.Kaa;
+import org.kaaproject.kaa.client.KaaClient;
+import org.kaaproject.kaa.client.notification.UnavailableTopicException;
+import org.kaaproject.kaa.schema.example.Notification;
+import org.kaaproject.www.kaanotification.R;
+
+public class KaaNotificationApp extends Application {
+
+    private static Context mContext;
+    private KaaClient mClient;
+
+    public void onCreate() {
+        super.onCreate();
+        mContext = this;
+        mClient = Kaa.newClient(new AndroidKaaPlatformContext(this));
+        initPopup();
```

Comment:
is there a reason why this is before client.start()? If not - move after start please.